src/Geometry_TEST.cc

```diff
@@ -395,6 +416,37 @@ TEST(DOMGeometry, CalculateInertial)
     EXPECT_EQ(expectedInertial.Pose(), capsuleInertial->Pose());
   }
 
+  // Cone
+  {
+    sdf::Cone cone;
+    const double l = 2.0;
+    const double r = 0.1;
+
+    cone.SetLength(l);
+    cone.SetRadius(r);
+
+    expectedMass = cone.Shape().Volume() * density;
+    double ixxIyy = (3 / 80.0) * expectedMass * (4 * r * r + l * l);
+    double izz = 3.0 * expectedMass * r * r / 10.0;
+
+    expectedMassMat.SetMass(expectedMass);
+    expectedMassMat.SetDiagonalMoments(gz::math::Vector3d(ixxIyy, ixxIyy, izz));
+    expectedMassMat.SetOffDiagonalMoments(gz::math::Vector3d::Zero);
+
+    expectedInertial.SetMassMatrix(expectedMassMat);
+    expectedInertial.SetPose(gz::math::Pose3d({0, 0, l / 4.0, 0, 0, 0}));
```

Comment:
```suggestion
    expectedInertial.SetPose(gz::math::Pose3d({0, 0, -l / 4.0, 0, 0, 0}));
```
this is what happens when I make suggestions without testing them